lib/internal/cluster.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+// Used in tests.
+exports.handles = {};
```

Comment:
was going to suggest the same: https://code.google.com/p/v8/issues/detail?id=4518